code/modules/modular_computers/computers/item/processor.dm

```diff
@@ -12,9 +12,11 @@
 	///The modular computer MACHINE that hosts us.
 	var/obj/machinery/modular_computer/machinery_computer
 
-/obj/item/modular_computer/processor/UpdateDisplay()
+/obj/item/modular_computer/processor/update_name()
 	. = ..()
-	//update our name to match the computer's
+
+	//update our name to match the computer's name
+	//blatantly ignore the COMSIG_ATOM_UPDATE_NAME call, this item is never visible anyways
```

Comment:
What do you mean by visible here? Examining the console should be returning the examine of the processor

Reply:
This only concerned a previous commit where I had `update_name()` code on `/obj/item/modular_computer` level since the proc was previously called `UpdateDisplay()`. Since `UpdateDisplay()` is gone and the logic in it has been moved down lower, the processor's name should never update, and therefore this should be no longer necessary